build.sh

```diff
@@ -187,6 +187,19 @@ for target_json in `jq -c '.targets[]' configs/builds.json`; do
     idf.py -DIDF_TARGET="$target" -DSDKCONFIG_DEFAULTS="$idf_libs_configs" idf_libs
     if [ $? -ne 0 ]; then exit 1; fi
 
+    if [ "$target" == "esp32s3" ]; then
+        idf.py -DIDF_TARGET="$target" -DSDKCONFIG_DEFAULTS="$idf_libs_configs" srmodels_bin
```

Comment:
@me-no-dev - It sounds like CI is failing here... It works in my computer :-)

https://github.com/espressif/esp32-arduino-lib-builder/actions/runs/6776484660/job/18438277663?pr=145
```
-- Build files have been written to: /home/runner/work/esp32-arduino-lib-builder/esp32-arduino-lib-builder/build
ninja: error: unknown target 'srmodels_bin'
ninja failed with exit code 1, output of the command is in the /home/runner/work/esp32-arduino-lib-builder/esp32-arduino-lib-builder/build/log/idf_py_stderr_output_20115 and /home/runner/work/esp32-arduino-lib-builder/esp32-arduino-lib-builder/build/log/idf_py_stdout_output_20115
Error: Process completed with exit code 1.
```

Reply:
not nice. We need a proper workaround

Reply:
I'll try to figure out a way to fix CI.

Reply:
@me-no-dev - I found out what is the issue here.
In order to add SR to this current PR, I need that the PR #8842 is merged, because it adds SR to the Arduino yml file.
After SR if int the managed components, it will run CI corretly here at Lib Builder CI.

As a way to test it, I've added SR to the local Lib Bulder yml file.

Reply:
Now it shows an Arduino esp32-hal-sr.c error:

-c /home/runner/work/esp32-arduino-lib-builder/esp32-arduino-lib-builder/components/arduino/libraries/ESP_SR/src/esp32-hal-sr.c
/home/runner/work/esp32-arduino-lib-builder/esp32-arduino-lib-builder/components/arduino/libraries/ESP_SR/src/esp32-hal-sr.c:10:2: warning: #warning Compatible partition must be selected for ESP_SR to work [-Wcpp]
   10 | #warning Compatible partition must be selected for ESP_SR to work
      |  ^~~~~~~
/home/runner/work/esp32-arduino-lib-builder/esp32-arduino-lib-builder/components/arduino/libraries/ESP_SR/src/esp32-hal-sr.c:27:10: fatal error: esp_mn_speech_commands.h: No such file or directory
   27 | #include "esp_mn_speech_commands.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.

Reply:
My suggestion is that we merge Arduino https://github.com/espressif/arduino-esp32/pull/8842 and then check how CI will work here at Lib Builder.

Reply:
https://github.com/espressif/arduino-esp32/actions/runs/6800936891/job/18490657761?pr=8842

Specific CI for the S3 is passing fine in the Arduino PR #8842
It shall run fine here as well.